src/Core/src/Core.csproj

```diff
@@ -1,19 +1,22 @@
-<Project Sdk="MSBuild.Sdk.Extras">
-	<PropertyGroup>
-		<TargetFrameworks>netstandard2.1;netstandard2.0;$(NonNet6Platforms)</TargetFrameworks>
-		<RootNamespace>Microsoft.Maui</RootNamespace>
-		<AssemblyName>Microsoft.Maui</AssemblyName>
-		<Nullable>enable</Nullable>
-	</PropertyGroup>
-	<PropertyGroup>
-		<PackageId>Microsoft.Maui</PackageId>
-	</PropertyGroup>
-	<Import Project="..\..\..\.nuspec\Microsoft.Maui.Controls.MultiTargeting.targets" />
-	<ItemGroup>
-		<PackageReference Include="Microsoft.Extensions.Configuration" />
-		<PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" />
-		<PackageReference Include="Microsoft.Extensions.Logging.Abstractions" />
-		<PackageReference Include="Microsoft.Maui.Graphics" />
-    	<PackageReference Include="Reloadify3000" Version="1.0.6" />
-	</ItemGroup>
+﻿<Project Sdk="MSBuild.Sdk.Extras">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.1;netstandard2.0;$(NonNet6Platforms)</TargetFrameworks>
+    <RootNamespace>Microsoft.Maui</RootNamespace>
+    <AssemblyName>Microsoft.Maui</AssemblyName>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+  <PropertyGroup>
+    <PackageId>Microsoft.Maui</PackageId>
+  </PropertyGroup>
+  <Import Project="..\..\..\.nuspec\Microsoft.Maui.Controls.MultiTargeting.targets" />
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Configuration" />
+    <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" />
+    <PackageReference Include="Microsoft.Maui.Graphics" />
+    <PackageReference Include="Reloadify3000" Version="1.0.6" />
+  </ItemGroup>
+  <ItemGroup Condition="$(TargetFramework.StartsWith('MonoAndroid'))">
+    <PackageReference Include="Xamarin.Android.Glide" />
```

Comment:
Out of curiosity, what is the size of Glide library?

Reply:
1.8MB when the aar is embedded in the dll. About 767KB of jar files.